BZ-12821: feat: Add right button css variables for input button component

prajwal-jp opened the pull request:
- Introduced CSS variables for the right button in the input button component
- Added a new parameter button-content-flex-direction to control the flex direction within  button component

<img width="168" alt="Screenshot 2024-06-18 at 5 13 59 PM" src="https://github.com/juspay/svelte-ui-components/assets/153722046/e13446ac-aca1-43d5-8c00-457543c18def">
<img width="135" alt="Screenshot 2024-06-18 at 5 14 31 PM" src="https://github.com/juspay/svelte-ui-components/assets/153722046/32d7927a-e411-415a-97d4-b9812b4cf5b4">

sandeepsj commented on src/lib/InputButton/InputButton.svelte:
```diff
@@ -129,12 +129,6 @@
 
-  .button {
-    flex: 1;
-    min-width: 0px;
```
is it safe to remove this min-width in new class?

prajwal-jp replied:
Adding it back as it might have unseen usecases

wardhan28 commented on src/lib/Button/properties.ts:
```diff
@@ -1,10 +1,12 @@
   enable: boolean;
   showLoader: boolean;
   loaderType: LoaderType | null;
+  circularLoaderPosition: CircularLoaderPosition | null;
```
can we make this field not nullable and initialise with 'start' ?

prajwal-jp committed:
BZ-12821: feat: Add right button css variables for input button widget

- Introduced CSS variables for the right button in the input button component
- Added a new parameter button-content-flex-direction to control the flex direction within  button component
